### PR TITLE
Update monitoring documentation.

### PR DESCRIPTION
**Pull request by r0mant**

### CODE REVIEW

File: docs/6.x/monitoring.md [hunk 1]
```diff
@@ -1,46 +1,67 @@
...
+native monitoring system and time-series database.
 
-### InfluxDB
+Prometheus uses the following in-cluster services to collect the metrics about the cluster:
 
-InfluxDB is the main data store for current + future monitoring time series data. It provides the
-service `influxdb.monitoring.svc.cluster.local`.
+* [node-exporter](https://github.com/prometheus/node_exporter) collects hardware and OS metrics.
+* [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) collects metrics about
+various Kubernetes objects such as deployments, nodes and pods.
```
Comment (a-palchikov) [inline docs/6.x/monitoring.md]: nit: Kubernetes objects -> Kubernetes resources

File: docs/6.x/monitoring.md [hunk 2]
```diff
@@ -1,46 +1,67 @@
...
+
+!!! note:
+    Collected metrics are stored for 30 days.
+
+Prometheus exposes the cluster-only service `prometheus-k8s.monitoring.svc.cluster.local:9090`.
```
Comment (a-palchikov) [inline docs/6.x/monitoring.md]: `cluster-internal`?

File: docs/6.x/monitoring.md [hunk 3]
```diff
@@ -1,46 +1,67 @@
...
-### Kapacitor
+Alertmanager exposes the cluster-only service `alertmanager-main.monitoring.svc.cluster.local:9093`.
 
-Kapacitor is the alerting system that streams data from InfluxDB and sends alerts as configured by
-the end user. It exposes the service `kapacitor.monitoring.svc.cluster.local`.
+## Grafana Integration
 
-## Grafana integration
+The default Grafana configuration includes two pre-configured dashboards providing machine- and
+pod-level overview of the installed cluster by default. Grafana UI is integrated with Gravity
```
Comment (a-palchikov) [inline docs/6.x/monitoring.md]: of the installed cluster ~~by default~~ (The sentence already hints at the default configuration).

File: docs/6.x/monitoring.md [hunk 4]
```diff
@@ -1,46 +1,67 @@
...
+control panel. To view dashboards once the cluster is up and running, navigate to the Cluster's
+Monitoring page.
 
-The standard Grafana configuration includes two pre-configured dashboards providing machine- and pod-level overview
-of the installed cluster by default. The Grafana UI is integrated with Gravity control panel. To view dashboards once
-the cluster is up and running, navigate to the Cluster's Monitoring page.
+By default, Grafana is running in anonymous read-only mode which allows anyone logged into Gravity
+to view existing dashboards (but not modify them or create new ones).
```
Comment (a-palchikov) [inline docs/6.x/monitoring.md]: This sentence begs the answer to a question about how make it run in the read-write mode and require login to view/update?

File: docs/6.x/monitoring.md [hunk 5]
```diff
@@ -1,46 +1,67 @@
...
 
-## Pluggable dashboards
+Your applications can create their own Grafana dashboards using ConfigMaps.
 
-Your applications can use their own Grafana dashboards using ConfigMaps.
-A custom dashboard ConfigMap should be assigned a `monitoring` label with value `dashboard` and created in the
-`monitoring` namespace so it is recognized and loaded when installing the application:
+A custom dashboard ConfigMap should be assigned a `monitoring` label with value `dashboard` and
+created in the `monitoring` namespace so it is recognized and loaded during initial cluster image
+installation:
```
Comment (a-palchikov) [inline docs/6.x/monitoring.md]: A custom dashboard ConfigMap needs to be placed into the `monitoring` namespace and assigned the special `monitoring:dashboard` label so it is recognized as a dashboard.

File: docs/6.x/monitoring.md [hunk 6]
```diff
@@ -59,119 +80,17 @@ Dashboard ConfigMap may contain multiple keys with dashboards and key names are
...
-* medium = 4w
-* long = 52w
-
-The `default` retention policy is supposed to store high-precision metrics (for example, all default metrics collected
-by Heapster with 10s interval). The `default` policy is default for `k8s` database which means that metrics that do not
-specify retention policy explicitly go in there. The other two policies - `medium` and `long` are intended to store
-metric rollups and should not be used directly.
+!!! tip:
+    Since the embedded Grafana runs in read-only mode, you can use a separate Grafana instance
+    to create a custom dashboard and then export its JSON.
```
Comment (a-palchikov) [inline docs/6.x/monitoring.md]: `and then export it as JSON` (or is anything else meant by `its JSON`?)

File: docs/6.x/monitoring.md [hunk 7]
```diff
@@ -180,60 +99,89 @@ metadata:
...
+   # email address of the alerts recipient
+  email: triage@example.com
 ```
 
-```bsh
-$ gravity resource create -f smtp.yaml
+Create the target:
+
+```bash
+$ gravity resource create -f target.yaml
```
Comment (a-palchikov) [inline docs/6.x/monitoring.md]: nit: `-f` will overwrite the existing configuration - should this be used as an example of introducing a resource? (Also it's confusing as `kubectl -f` specifies a filename, so I would use the full format: `gravity resource create --force target.yaml`).

File: docs/6.x/monitoring.md [hunk 8]
```diff
@@ -180,60 +99,89 @@ metadata:
...
   formula: |
-    Kapacitor formula
+    node:cluster_cpu_utilization:ratio * 100 > 80
+  # the alert labels
+  labels:
+    severity: info
+  # the alert annotations
+  annotations:
+    description: |
+      This is a first test alert
```
Comment (a-palchikov) [inline docs/6.x/monitoring.md]: nit: this should ideally be consistent with the example, i.e. `Alert when CPU utilization exceeds 80%`

**Comment by twakes** [comment]
The one general comment I have is why don't we include information about [Satellite](https://github.com/gravitational/satellite) in this section? I think an overview of what it is and why it is included with a link to the repo would be helpful.

A smaller comment is that I have been capitalizing Cluster Image(s) and Cluster(s) elsewhere in the docs but I can also clean that up post-merge.

**Pull request merged**